### DISCUSSION TITLE
Mixtral 8x7B support?

### DISCUSSION THREAD

**Discussion by clintg6** [erfanzar/EasyDeL#85] [Q&A] [answered]
Hi Erfan, I tried testing the Mixtral module and it failed returning "Returning Empty Params". Is this module working?

**Comment by erfanzar** [comment, marked as answer]
Hello due to not being able to evaluate the model params mixtral model can not be traced and that's a jax issue but if you load the model from torch or easy state you can ignore that cause model is loaded and you can use that.
